cypress/e2e/functional/researcher_permission_forms.spec.js

```diff
@@ -0,0 +1,113 @@
+import * as c from '../../support/constants.js'
+import {ResearcherPermissionFormsElements as pf} from "../../support/elements/researcher_permission_forms_elements";
+
+const TEST_PROJECT_NAME = 'Test Project',
+    TEST_PERMISSION_FORM_NAME = 'PermForm4CypressTests',
+    TEST_PERMISSION_FORM_URL  = 'concord.org'
+    // TODO: I don't know the password for c.RESEARCHER_USERNAME (i.e. 'researcher'), so I've created my own, for now
```

Comment:
We should get this information from @saravanan-udayakumar if possible.

Reply:
@nstclair-cc @johnTcrawford for all our automation test data we follow the common password called **password**.

Reply:
Thank you, Saravanan! Although I deleted your comment, since this is an open GitHub repo.